Sources/Testing/ABI/EntryPoints/EntryPoint.swift

```diff
@@ -628,15 +627,15 @@ public func configurationForEntryPoint(from args: __CommandLineArguments_v0) thr
 #endif
 
   // Warning issues (experimental).
-  switch args.eventStreamVersionNumber {
-  case .some(..<ABI.v6_3.versionNumber):
-    // If the event stream version was explicitly specified to a value < 6.3,
-    // disable the warning issue event to maintain legacy behavior.
-    configuration.eventHandlingOptions.isWarningIssueRecordedEventEnabled = false
-  default:
-    // Otherwise the requested event stream version is ≥ 6.3, so don't change
-    // the warning issue event setting.
-    break
+    switch args.eventStreamVersionNumber {
+    case .some(..<ABI.v6_3.versionNumber):
+      // If the event stream version was explicitly specified to a value < 6.3,
+      // disable the warning issue event to maintain legacy behavior.
+      configuration.eventHandlingOptions.isWarningIssueRecordedEventEnabled = false
+    default:
+      // Otherwise the requested event stream version is ≥ 6.3, so don't change
+      // the warning issue event setting.
+      break
```

Comment:
nit: I think the indentation here was unintentional, probably the result of a merge conflict resolution where these lines were recently modified. You can "outdent" them one level.